src/main/java/com/liveramp/captain/notifier/DefaultCaptainLoggingNotifier.java

```diff
@@ -7,9 +7,21 @@
 public class DefaultCaptainLoggingNotifier implements CaptainNotifier {
   private static final Logger LOG = LoggerFactory.getLogger(DefaultCaptainLoggingNotifier.class);
 
-
   private void notifyInternal(String subject, String body, NotificationLevel notificationLevel) {
-    LOG.info(String.format("[%s]: %s %n%n %s)", notificationLevel.toString(), subject, body));
+    String msg = String.format("[%s]: %s %n%n %s)", notificationLevel.toString(), subject, body);
+    switch (notificationLevel) {
```

Comment:
make logs easier to search by using the notification level to know which type of log we should use.